build.sh

```diff
@@ -53,7 +53,7 @@ cd service
 ./autogen.sh
 mkdir -p build
 cd build
-../configure ${GEOPM_SERVICE_CONFIG_OPTIONS}
+CC=gcc CXX=g++ ../configure ${GEOPM_SERVICE_CONFIG_OPTIONS}
```

Comment:
We might want to do something in the future like:
```suggestion
CC=/usr/bin/gcc CXX=/usr/bin/g++ ../configure ${GEOPM_SERVICE_CONFIG_OPTIONS}
```
to make sure that compilers coming in with module loads don't accidentally get used here.